airbyte-webapp-e2e-tests/cypress/pages/connectorBuilderPage.ts

```diff
@@ -94,3 +123,12 @@ export const testStream = () => {
   cy.wait(500);
   cy.get(testStreamButton).click();
 };
+
+const GO_BACK_AND_GO_NEXT_BUTTONS = 2
+export const assertHasNumberOfPages = (numberOfPages: number) => {
+  for (var i = 0; i < numberOfPages; i++) {
+    cy.get(testPageItem).contains(i + 1).should('exist')
+  }
+
+  cy.get(testPageItem).should('have.length', numberOfPages + GO_BACK_AND_GO_NEXT_BUTTONS)
```

Comment:
I feel this assertion is a bit weird but the for loop is not sufficient since there could be more elements that exist. However, if the arrows placement is changed, this test will fail

Reply:
See comment above, this way you don't need this assertion

Reply:
@maxi297 can you add a comment saying that
> if the arrows placement is changed, this test will fail